lib/mongodb.js

```diff
@@ -802,6 +821,49 @@ MongoDB.prototype.count = function count(model, where, options, callback) {
 };
 
 /**
+ * Replace properties for the model instance data
+ * @param {String} model The model name
+ * @param {*} id The instance id
+ * @param {Object} data The model data
+ * @param {Object} options The options object
+ * @param {Function} [cb] The callback function
+ */
+MongoDB.prototype.replaceAttributes = function replaceAttrs(model, id, data, options, cb) {
```

Comment:
That's a good question; please see this [comment](https://github.com/strongloop/loopback-datasource-juggler/pull/788#issuecomment-164428158)